src/components/TaskItem.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const TaskItem = (props) => (
+        <li>TASK NAME : {props.task.name}</li>
+);
+
+TaskItem.propTypes = {
+    task: React.PropTypes.object.isRequired,
+};
+
```

Comment:
Maybe you could change this line of code to avoid deprecated-notification in console. 
https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html